### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by hugovk**
* Update Black and devguide link redirects
* Update badge: `master` is now `main`
* Code formatting and wording for test instructions

### CODE REVIEW

File: readme.rst [hunk 1]
```diff
@@ -61,11 +61,10 @@ Requirements and dependencies
...
-Running Tests
+Running tests
 =============
 
-1. Install pyhon virtual environment with `$ python3 -m venv venv`.
-2. Activate virtual environment with `$ . venv/bin/activate`.
-3. Install dev-requirements with `(venv)$ pip install -r dev-requirements.txt`.
-4. Run all test with `(venv)$ pytest tests`.
-
+1. Install a Python virtual environment with ``$ python3 -m venv venv``
```
Comment (JelleZijlstra) [inline readme.rst:67]:
```suggestion
1. Create a Python virtual environment with ``$ python3 -m venv venv``
```
"Install" is a strange choice of words here

**Review by JelleZijlstra — APPROVED** [review]

File: readme.rst [hunk 2]
```diff
@@ -19,8 +18,9 @@ About
 📜🤖 blurb-it allows you to add a misc/news file to your own
 `CPython <https://github.com/python/cpython>`_ pull request.
 
-Misc/News file `is needed <https://devguide.python.org/committing/#what-s-new-and-news-entries>`_ for
-almost all non-trivial changes to CPython.
+A Misc/News file `is needed
```
Comment (ezio-melotti) [inline readme.rst:21]:
```suggestion
A ``Misc/News`` file `is needed
```

**Review by ezio-melotti — APPROVED** [review]

File: readme.rst (hunk 2 above)
Comment (hugovk) [inline readme.rst:21]: Or?
```suggestion
A ``Misc/NEWS.d`` file `is needed
```

Re: https://github.com/python/cpython/tree/main/Misc/NEWS.d

**Comment by hugovk** [comment]
Close/re-open to trigger CI, for some reason it didn't run for the last commit.

**Pull request merged**